### PR TITLE
Added extended xkcd font, support hidden position, example

### PR DESCRIPTION
**Pull request by tonyrewin**

### CODE REVIEW

**Comment by tonyrewin** [comment]
font is converted from the original from here: https://www.reddit.com/r/xkcd/comments/6tbt88/i_added_cyrillic_and_a_bunch_of_extended_latin_to/

**Comment by shiyiya** [comment]
you can fix #15

**Comment by timqian** [comment]
Thanks for the PR @tonyrewin the extended xkcd font is great but there are several issues in the code
- .DS_Store file is still included in the PR, you should delete the file first then ignore it
- the font URL seems won't work
- to hide the legend, there are bugs in the console, in the current implementation, we will need to check the legendPosition in `Line.js`, `Pie.js`, `XY.js`.

The current way to display legend is not so good. It may on top of data, I have the plan to refactor it to support move the legend out of the graph part

File: src/utils/addFont.js [hunk 1]
```diff
@@ -4,6 +4,6 @@ export default function addFont(parent) {
     .attr('type', 'text/css')
     .text(`@font-face {
       font-family: "xkcd";
-      src: url('https://cdn.rawgit.com/ipython/xkcd-font/master/xkcd-script/font/xkcd-script.woff');
+      src: url('/fonts/xkcd-script-extended.woff');
```
Comment (tonyrewin) [inline src/utils/addFont.js]: is not an url to rawgit anymore, is distributed font file here

**Pull request closed without merging**